polymorphic-linux-install-alpine-container/reinstall.md

```diff
@@ -1,6 +1,6 @@
 Great work, you are now subscribed to the Polyverse mirrors. This new repository is completely transparent to you, and does not change how you use your package manager.
 
-In order to get the full power of Polymorphic Linux reinstall all packages. For Alpine, lets run:
+In order to get the full power of Polymorphing reinstall all packages. For Alpine, lets run:
```

Comment:
Do you think this would be better with a comma?
```
In order to get the full power of Polymorphing, reinstall all packages.
```

Reply:
Going to merge it in anyway. You may fix it later if it feels reasonable.

Reply:
Yes that should have a comma